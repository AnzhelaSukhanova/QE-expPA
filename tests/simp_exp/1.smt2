(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (or (bvult (bvshl #b0001 x) y) (= (bvshl #b0001 x) y))))
