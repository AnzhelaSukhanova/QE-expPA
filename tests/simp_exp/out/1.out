(let (($e1 (= y #x0))) (let (($e2 (ite (bvult y #x8) (ite (bvult y #x4) (ite (bvult y #x2) (ite $e1 #xf #x0) #x1) #x2) #x3))) (let (($e3 (= y #x1))) (let (($e4 (= y #x2))) (let (($e5 (= y #x4))) (let (($e6 (= y #x8))) (let (($e7 (bvult #x1 y))) (let (($e8 (bvult #x2 y))) (let (($e9 (bvult #x4 y))) (let (($e10 (bvult #x8 y))) (not (and (not $e8) (not $e7) (not $e9) (not $e4) (not $e3) (not $e10) (not $e5) $e1 (not $e6) (not $e7) (not $e1) (not $e8) (not $e3) (not $e9) (not $e4) (not $e10) (not $e5) $e1 (not $e6) (not $e7) (not $e1) (not (bvult (bvshl #x1 (bvadd #x1 $e2)) y)) (not (bvult (bvshl #x1 $e2) y)) (not $e8) (not $e3) (not $e4)))))))))))))
